### DISCUSSION TITLE
Search tool for blog articles

### DISCUSSION THREAD

**Discussion by Dublinscope** [arthelokyo/astrowind#27] [Ideas]
Hi,

Please could you consider adding a search for blog articles ?
There is an open source solution called Lunr.
https://lunrjs.com/
https://github.com/olivernn/lunr.js

All the best.

**Comment by prototypa** [comment]
Hello @Dublinscope ,

I've thought about it but I hadn't prioritized it. However it makes a lot of sense in a Blog.

I was looking that there is an integration of Lunr.js with Astro, although it is still in very unstable versions: https://www.npmjs.com/package/@siverv/astro-lunr

I'm going to be reviewing the easy way to add that to the template but make it optional and configurable. If you have already done so, PRs are welcome. 
Thank you
